### PR TITLE
📖 [Amp story] [Page attachments] Amp.dev Docs for New Page Attachment Features #34883

### LINKED ISSUE #34790: [Page Attachments] Write amp.dev docs
### Description

Write updated page attachment docs for amp.dev

### Alternatives Considered

Largely pulling from IS2, need to be ready for Canary launch

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by processprocess**
Updates `amp-story-page-attachment` docs
Adds `amp-story-page-outlink` docs

Fixes #34790

### CODE REVIEW

File: extensions/amp-story/amp-story-page-attachment.md [hunk 1]
```diff
@@ -59,32 +59,109 @@ A UI prompt to open the attachment will automatically be added at the bottom of
...
+<amp-story-page-attachment
+  layout="nodisplay"
+  cta-text="Call To Action">
+   ...
+</amp-story-page-attachment>
+```
+
+### `cta-image` (optional)
+
+String representing a URL pointing to an image. 48x48px is suggested.
```
Comment (CrystalOnScript) [inline extensions/amp-story/amp-story-page-attachment.md:113]: I'm a little stuck on the word "suggested" - will other sizes work? I would update the language to something like "Optimize performance and avoid distortion by using a 48x48px image"

File: extensions/amp-story/amp-story-page-attachment.md [hunk 2]
```diff
@@ -59,32 +59,109 @@ A UI prompt to open the attachment will automatically be added at the bottom of
...
+   ...
+</amp-story-page-attachment>
+```
+
+![amp-story-page-attachment-1-image](img/amp-story-page-attachment-1-image.jpg)
+
+### `cta-image-2` (optional)
 
-When providing a `href` attribute as page attachment, the respective URL is opened when the user activates the page attachment.
+String representing a URL pointing to an image. 48x48px is suggested. Two images will display when `cta-image` and `cta-image-2` are defined.
```
Comment (CrystalOnScript) [inline extensions/amp-story/amp-story-page-attachment.md:127]: Just wanted to highlight "suggested" here.

File: extensions/amp-story/amp-story-page-outlink.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  theme="custom"
+  cta-accent-element="text"
+  cta-accent-color="#247C3C">
+    <a href="https://www.google.com">Call To Action</a>
+</amp-story-page-outlink>
+```
+
+### `cta-image` (optional)
+
+String representing a URL pointing to an image. 32x32px is suggested. A link icon will be displayed by default.
```
Comment (CrystalOnScript) [inline extensions/amp-story/amp-story-page-outlink.md:135]: And "suggested" here

**Review by CrystalOnScript — APPROVED** [review]
Approved - with one nit to change the use of the word "suggested". Thanks!

**Comment by MCFreddie777** [comment]
@processprocess  I think you removed `## Linked content` section by an accident.

**Comment by processprocess** [comment]
> @processprocess I think you removed `## Linked content` section by an accident.

Hi @MCFreddie777 :) 
We created the [amp-story-page-outlink](https://amp.dev/documentation/components/amp-story-page-outlink/?format=stories) component for linking behavior. We did this so that amp-story-page-attachment can handle all inline attachment logic and amp-story-page-outlink can handle all outlink logic. 
We will be posting a blogpost soon that explains all of this in more detail. 
Sorry for any confusion!

**Comment by processprocess** [comment]
@MCFreddie777 The blogpost with details on the CTA updates is available [here](https://blog.amp.dev/2021/09/15/web-stories-call-to-action-buttons-get-an-upgrade/) :)

**Pull request merged**